apps/gimbtonomy/main.cpp

```diff
@@ -8,71 +8,78 @@
 #include "rover_config.h"
 #include "Neopixel_Blocking.h"
 
-Serial        pc(SERIAL_TX, SERIAL_RX, ROVER_DEFAULT_SERIAL_BAUD_RATE);
+constexpr uint16_t    NEOPIXEL_CAN_ID_INCOMING = 0x784;
+constexpr uint16_t    NEOPIXEL_CAN_ID_OUTGOING = 0x785;
+CAN                   can(CAN1_RX, CAN1_TX, ROVER_CANBUS_FREQUENCY);
+CANMsg                rxMsg, txMsg;
 // CAN_RX = PB_8, CAN_TX = PB_9
-CAN           can(CAN1_RX, CAN1_TX, ROVER_CANBUS_FREQUENCY);
-CANMsg        rxMsg;
 Neopixel_Blocking      neopixel(16, LED_MTRX);
 
-// 0x794 CAN ID for change to a neo pixel.
 // Color is specified by the data inside the packet
 // 0 is solid red
 // 1 is solid blue
 // 2 is flashing green
+// 3 is off
```

Comment:
instead of comments, use enum or defines

Reply:
yup i agree. should have an enum here